huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_pg_account_test.go

```diff
@@ -0,0 +1,147 @@
+	}
+
+	getPgAccountRespJson, err := json.Marshal(getPgAccountResp)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving RDS PostgreSQL account: %s", err)
+	}
+
+	var getPgAccountRespBody interface{}
+	err = json.Unmarshal(getPgAccountRespJson, &getPgAccountRespBody)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving RDS PostgreSQL account: %s", err)
+	}
+
+	account := utils.PathSearch(fmt.Sprintf("users[?name=='%s']|[0]", accountName), getPgAccountRespBody, nil)
+
+	if account != nil {
+		return account, nil
+	}
+
+	return getPgAccountRespBody, nil
+}
+
+func TestAccPgAccount_basic(t *testing.T) {
+	var obj interface{}
+
+	name := acceptance.RandomAccResourceName()
+	updateName := acceptance.RandomAccResourceName()
+	rName := "huaweicloud_rds_pg_account.test"
+	pwd := fmt.Sprintf("%s%s%d", acctest.RandString(5), acctest.RandStringFromCharSet(2, "!#%^*"),
+		acctest.RandIntRange(10, 99))
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getPgAccountResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testPgAccount_basic(name, pwd),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(rName, "instance_id",
+						"huaweicloud_rds_instance.test", "id"),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttr(rName, "description", "test_description"),
+				),
+			},
+			{
+				Config: testPgAccount_basic_update(updateName, pwd),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(rName, "instance_id",
+						"huaweicloud_rds_instance.test", "id"),
+					resource.TestCheckResourceAttr(rName, "name", updateName),
+					resource.TestCheckResourceAttr(rName, "description", "test_description_update"),
+				),
+			},
+			{
+				ResourceName:            rName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"password"},
+			},
+		},
+	})
+}
+
+func testPgAccount_basic(name, dbPwd string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_rds_pg_account" "test" {
+  instance_id = huaweicloud_rds_instance.test.id
+  name        = "%s"
+  password    = "Test@12345678"
+  description = "test_description"
+}
+`, testAccRdsInstance_basic(name, dbPwd), name)
+}
+
+func testPgAccount_basic_update(name, dbPwd string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_rds_pg_account" "test" {
+  instance_id = huaweicloud_rds_instance.test.id
+  name        = "%s"
+  password    = "Test@123456789"
+  description = "test_description_update"
+}
+`, testAccRdsInstance_basic(name, dbPwd), name)
+}
```

Comment:
```suggestion
func testPgAccount_basic(name, dbPwd,accountPwd, description string) string {
	return fmt.Sprintf(`
%s

resource "huaweicloud_rds_pg_account" "test" {
  instance_id = huaweicloud_rds_instance.test.id
  name        = "%s"
  password    = "Test@12345678"
  description = "test_description"
}
`, testAccRdsInstance_basic(name, dbPwd), name,accountPwd, description)
}
```

Reply:
done

Reply:
make testacc TEST=./huaweicloud/services/acceptance/rds/ TESTARGS='-run TestAccPgAccount_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/rds/ -v -run TestAccPgAccount_basic -timeout 360m -parallel 4
=== RUN   TestAccPgAccount_basic
=== PAUSE TestAccPgAccount_basic
=== CONT  TestAccPgAccount_basic
--- PASS: TestAccPgAccount_basic (624.37s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/rds       624.415s